ui/src/components/settings/styles.scss

```diff
@@ -51,6 +52,13 @@
         flex-direction: column;
         gap: var(--gap-less);
     }
+    .settings-container {
+        flex: 1;
+        min-width: 0;
+        display: inline-flex;
+        flex-direction: row;
+        gap: var(--gap-less);
+    }
```

Comment:
instead of creating this, we should reuse the existing styles for the settings stuff if possible

Reply:
it would be easier if there were CSS classes for flexboxes so that I could make something display as flex-column without having to make a custom class for the element. Should I make that? Besides that, I don't see a better option than what's currently there.

Reply:
done